Source/ThirdParty/skia/CMakeLists.txt

```diff
@@ -0,0 +1,1019 @@
+set_property(DIRECTORY . PROPERTY FOLDER "skia")
+
+if (SKIA_ROOT_DIR OR NOT "$ENV{SKIA_ROOT_DIR}" STREQUAL "")
+    if (NOT SKIA_ROOT_DIR)
+        set(SKIA_ROOT_DIR "$ENV{SKIA_ROOT_DIR")
+    endif ()
```

Comment:
This part old? Looks like it was supporting a `SKIA_ROOT_DIR` out of source.

Reply:
We were debating whether to remove this or not, without much of a conclusion. Personally I would prefer it gone; but others have found it useful during development.

@alexgcastro, @nikolaszimmermann and @carlosgcampos: Could we remove this, or is there still any good reason why we would need to keep it a bit longer?

Reply:
I would keep it for now. We are still seeing differences in some cases and it's very convenient to be able to try with external skia build to compare. We can remove it once we are sure we are building skia with the right options.

Reply:
We can add a FIXME explaining this is temporary and will be removed eventually.

Reply:
Yeah, please keep it, I think it is very valuable even in the future to check if the compilation we are doing with CMake provides the same results and performance, it simplifies testing the original gn compilation and compare the results.

Reply:
Agreed. Let us keep it for now and add a FIXME.

Reply:
I'll add a comment about why we are keeping it for now, and also gate it behind `DEVELOPER_MODE` so that way it is clear that the intention is to have it as an aid for developers.